docs/source/notebooks/model_comparison.ipynb

```diff
@@ -1,25 +1,34 @@
 {
```

Comment:
<p>Shouldn't you call ArviZ directly in these last two cells? Especially since you're talking about ArviZ in the text</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/4042/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>